python/python/glide/async_commands/core.py

```diff
@@ -2662,6 +2664,72 @@ async def zmscore(
             await self._execute_command(RequestType.ZMScore, [key] + members),
         )
 
+    async def zinterstore(
+        self,
+        destination: str,
+        keys: Union[List[str], List[Tuple[str, int]]],
+        aggregation_type: Optional[AggregationType] = None,
+    ) -> int:
+        """
+        Computes the intersection of sorted sets given by the specified `keys` and stores the result in `destination`.
+        If `destination` already exists, it is overwritten. Otherwise, a new sorted set will be created.
+        When in cluster mode, `destination` and all `keys` must map to the same `hash slot`.
+        see https://redis.io/commands/zinterstore/ for more details.
```

Comment:
```suggestion
        See https://valkey.io/commands/zinterstore/ for more details.

        When in cluster mode, `destination` and all `keys` must map to the same hash slot.
```
Thanks for adding cross slot notice